examples/weather-demo/Cargo.toml

```diff
@@ -0,0 +1,68 @@
+[package]
+name = "rusty-weather"
+version = "0.1.0"
+authors = ["FELGO GmbH <contact@felgo.com>"]
+edition = "2021"
+build = "build.rs"
+
+[dependencies]
+async-std = "1.12.0"
+chrono = "0.4.38"
+directories = "5.0.1"
+log = "0.4.21"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.115"
+
+[target.'cfg(all(not(target_os = "windows"), not(target_os = "android")))'.dependencies]
+slint = { path = "../../api/rs/slint" }
+
+# Using different Slint configuration on Windows to disable the accessibility feature (enabled by default).
+# The feature caused performance issues on Windows (see https://github.com/slint-ui/slint/issues/5338).
+# Note: might be fixed with v1.7
```

Comment:
Since this is being built with "1.7", does this still apply?

Reply:
I believe so since the ticket ([the one that mine was a duplicate of](https://github.com/slint-ui/slint/issues/5041)) is still open. But I can double-check this.

Reply:
Ok :). We can deal with this separately and leave this.

Reply:
Ok, I checked and this is fixed now in master AFAICS.

Reply:
I checked and can confirm that the issue is gone. Let me update the file.